doc/architectural-overview.md

```diff
@@ -106,10 +106,10 @@ responsible for installing, running and maintaining charm code. It runs a
 different set of workers to a machine agent.
 
 There are a number of *clients* which interact with Juju using the Juju
```

Comment:
when you say "interact with Juju" you really mean interact with the Juju controller agent using the Juju API?